File: fullstack-dev/rabbitmq/dotnet/Producser/Producer.cs
using System.Text;
using RabbitMQ.Client;
var queue = "ConcurrentConsumer";
var uri = new Uri("amqp://frappet:Password@localhost");
var factory = new ConnectionFactory(){Uri=uri};
using var connection = factory.CreateConnection();
using var channel = connection.CreateModel();
channel.QueueDeclare(
    queue,
    durable: false,
    exclusive: false,
    autoDelete: false,
    arguments: null);
Console.WriteLine(" Type Message to send (type exit to quit)");
var messageId = 1;
while(true)
{
    string line = Console.ReadLine()??"";
    if(line.Equals("exit")) break;
    var message = $"[{messageId}] {line}";
    var body = Encoding.UTF8.GetBytes(message);
    channel.BasicPublish(exchange: string.Empty, routingKey:queue, basicProperties:null, body: body);
    Console.WriteLine($"Send message: {message}");  
    messageId++;
}
channel.Close();
connection.Close();

File: fullstack-dev/rabbitmq/node/producer.js
import { connect } from "amqplib"
const queue = "concurrent-queue"
const connection =  await connect("amqp://oom:Password@localhost")
const channel = await connection.createChannel()
await channel.assertQueue(queue,{durable:true})
for(let i=0;i<10;i++){
    const message="Hello "+i
    channel.sendToQueue(queue,Buffer.from(message),{persistent: false})    
}
await channel.close()
await connection.close()